source/client/process_context_impl.h

```diff
@@ -25,6 +25,15 @@
 namespace Nighthawk {
 namespace Client {
 
+constexpr const char* ProcessContextLockFile = "/tmp/nighthawk.lock";
+
+/**
+ * Only a single instance is allowed at a time machine-wide in this implementation.
```

Comment:
This one seems a bit strong; can't we have the end user make the call if we give them best practices? I agree we should prevent obviously bad things, but I can imagine situations in which you want to load test without latency measurement and want multiple NHs.

Reply:
Yeah, so I was in two minds about this myself as well; would adding something like `--instance-name=unique-name` to allow distinct multiple concurrent instances resolve this in a way you would be comfortable with?

Reply:
Yep, sounds good. Maybe in a separate PR?

Reply:
SG, created https://github.com/envoyproxy/nighthawk/issues/87 for tracking that new option.